ipapython/ipa_log_manager.py

```diff
@@ -141,12 +160,45 @@ def filter(self, record):
                 record.levelno >= self.level)
 
 
+class InfoCommandFilter(logging.Filter):
+    """
+    The InfoCommandFilter filters out info_cmd log messages for the log file.
+    info_cmd log messages should only be visble on the command line.
+    """
+
+    def filter(self, record):
+        if hasattr(record, "cmd_output") and record.__dict__["cmd_output"]:
```

Comment:
This can be simplified to ``return getattr(record, "cmd_output", False)``. All extras are available as attributes on the record object.

Reply:
It can be simplified to `return not getattr(record, "cmd_output", False)`.